ngmt/test/test_calc.py

```diff
@@ -1682,28 +1690,28 @@ def test_tilt_angle_estimation():
 
 # Test function for gsd_plot_results without plotting
 def test_gsd_plot_results_without_plot(monkeypatch):
```

Comment:
What is monkeypatch?

Reply:
"monkeypatch" is a feature provided by the pytest library. 
It allows to temporarily modify or replace attributes, methods, or functions during testing. Here, monkeypatch is being used to replace the plt.show() function. This way, when run test test_gsd_plot_results_without_plot function, it won't actually display any plots, allowing to focus on testing the logic of the function without the overhead of rendering plots.